vunit/builtins.py

```diff
@@ -232,3 +241,6 @@ def _add_check(self, name, args=None):
                 "Optional builtin %r added with arguments %r has already been added with arguments %r"
                 % (name, args, old_args))
         return True
+
```

Comment:
This function does not need to be public in the Builtins class (Add _ to make private) in fact this funciton is not necessary at all.

Reply:
` Access to a protected member _get_types of a client class (protected-access)` 
Can you explain why is not necessary?

Reply:
Ok they where different classes so I was wrong. It needs to be public.